### PR TITLE
[Doc] Support download and ingest

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+```
+echo $HADOOP_HOME
+```
+
+### Download data
+
+```
+DOWNLOAD HDFS "hdfs://${HADOOP_HOST}:${HADOOP_PORT}/${HADOOP_PATH}"
+```
```
Comment (jude-zhu) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: `DOWNLOAD` statement will download the sst to /download dir. Could it be configured by users? How many threads are configured to download the files by default? Is it supposed to change by the users?

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Nebula use `DOWNLOAD` command to get `SST` files from `HDFS`.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: Nebula downloads `SST` files into the download directory with `DOWNLOAD` command.

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Please make sure the `Hadoop` have been installed and `HADOOP_HOME` have been set.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: please delete `the`

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Using `INGEST` to loading `SST` files which download from `HDFS`.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md]: Use `INGEST` to load `SST` files download from `HDFS`.

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md (hunk 1 above)
Comment (darionyaphet) [reply docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: currently default download thread is 3 . It configured by `download_thread_num`

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+请确认 `Hadoop` 已安装，且 `HADOOP_HOME` 已配置
+
+```
+echo $HADOOP_HOME
+```
+
+### 数据下载
+
+```
+DOWNLOAD HDFS "hdfs://${HADOOP_HOST}:${HADOOP_PORT}/${HADOOP_PATH}"
```
Comment (laura-ding) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: add `nebula>` indicates that the command is executed through the console.

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+Please make sure `Hadoop` have been installed and `HADOOP_HOME` have been set.
+
+```
+echo $HADOOP_HOME
+```
+
+### Download data
+
+```
+DOWNLOAD HDFS "hdfs://${HADOOP_HOST}:${HADOOP_PORT}/${HADOOP_PATH}"
```
Comment (laura-ding) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: ditto

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+Nebula 使用 `INGEST` 语句加载 通过 `DOWNLOAD` 语句下载的 `SST` 文件。
+
+### 数据加载
+
+```
+INGEST
```
Comment (laura-ding) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md]: ditto

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+Using `INGEST` to load `SST` files downloaded from `HDFS`.
+
+### Loading data
+
+```
+INGEST
```
Comment (laura-ding) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md]: ditto

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md (hunk 5 above)
Comment (darionyaphet) [reply docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: emmm... This the language syntax not an example should not running in console.

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md (hunk 7 above)
Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md]: 我直接在console里面输入 INGEST就可以吗？

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+Nebula downloads `SST` files into the download directory with `DOWNLOAD` command.
+
+Please make sure `Hadoop` have been installed and `HADOOP_HOME` have been set.
+
+```
+echo $HADOOP_HOME
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md]: 如果是shell，用符号>
如果是console, 用符号nebula >
不然可能会搞混。

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md (hunk 7 above)
Comment (darionyaphet) [reply docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/ingest-syntax.md]: 是的

**Comment by Amber1990Zhang** [comment]
LGTM

**Pull request merged**